### PR TITLE
add button and alert component

### PR DESCRIPTION
**Pull request by BISINIGISHIVANI**
add alert and button component

### CODE REVIEW

File: components/badge/index.html [hunk 1]
```diff
@@ -23,8 +23,8 @@
         <span class="badge-absolute badge-btn badge-bold badge-round badge-blue">23</span>
       </div>
       <div  class="badge-relative">
-        <button>Hide</button>
-        <span class="badge-absolute badge-btn-right badge-bold badge-round badge-blue">5</span>
+        <button onClick="myFunction()"id="btn-hide">Hide</button>
```
Comment (shaik-saad) [inline components/badge/index.html]: You can use DOM to add functionality to this button, using DOM to add a click listener to this function is a better practice.

File: components/badge/script.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+const inputOne=document.querySelector("#btn-ide");
```
Comment (shaik-saad) [inline components/badge/script.js]: You have given "#btn-ide" here, it's a typo. That's why your function was not working with the add event listener.

File: components/badge/script.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+function myFunction(){
+    const userInput=document.querySelector("#toggle1");
```
Comment (shaik-saad) [inline components/badge/script.js]: You can change the variable name and name it something relatable, as there is no user input here, it leads to confusion

**Review by shaik-saad — CHANGES_REQUESTED** [review]
Added some suggestions, do check them out.

File: components/badge/index.html (hunk 1 above)
Comment (BISINIGISHIVANI) [reply components/badge/index.html]: `const btnInput=document.querySelector("#btn-input");
const hideBadge=()=>{
    const badge=document.querySelector("#badge-hide");
    if(badge.style.visibility === "visible"){
        badge.style.visibility="hidden";
    }else{
        badge.style.visibility="visible";
    }
}
btnInput.addEventListener("click",hideBadge);`

Comment (BISINIGISHIVANI) [reply components/badge/index.html]: okay i just updated have a look it

File: components/badge/script.js (hunk 2 above)
Comment (BISINIGISHIVANI) [reply components/badge/script.js]: i change the name of it

File: components/badge/script.js [hunk 4]
```diff
@@ -1,11 +1,10 @@
-const inputOne=document.querySelector("#btn-ide");
-
-function myFunction(){
-    const userInput=document.querySelector("#toggle1");
-    if(userInput.style.visibility === "visible"){
-        userInput.style.visibility="hidden";
+const btnInput=document.querySelector("#btn-input");
+const hideBadge=()=>{
+    const badge=document.querySelector("#badge-hide");
```
Comment (shaik-saad) [inline components/badge/script.js:3]: you can create this reference at the starting of the document, it's a better practice I guess.

**Review by shaik-saad — APPROVED** [review]
Add one more suggestion, other than that you're good to merge

**Review by mohammed-sas — APPROVED** [review]
pr good to merge

**Pull request merged**